build/storage/scripts/disk_infrastructure.py

```diff
@@ -143,8 +143,8 @@ def create_ramdrive_and_attach_as_ns_to_subsystem(
             "method": "bdev_malloc_create",
             "params": {
                 "name": ramdrive_name,
-                "num_blocks": ramdrive_size_in_mb * 1024 * 1024 // 512,
-                "block_size": 512,
+                "num_blocks": ramdrive_size_in_mb * 256,
+                "block_size": 4096,
```

Comment:
Let's discuss this change more in a meeting.

Reply:
If we should need to specify block size for other targets, then it won't take much time to modify this code